### ISSUE TITLE
Garbage at end of spec

### ISSUE THREAD

**Issue by gagern** [jgm/CommonMark#387]
There is some garbage included at the end of [the 0.23 spec](http://spec.commonmark.org/0.23/):

    <p>After we’re done, we remove all delimiters above <code>stack_bottom</code> from the
    delimiter stack.</p>
    <p>[&gt;</p>
    <pre><code>
    
    &lt;/div&gt;
    
    &lt;div class=&quot;column&quot;&gt;
    
    ````````````````````````````````````````````````````````` html
    &lt;p&gt;&lt;a<span class="space"> </span>href=&quot;http://example.com/%5C%5B%5C&quot;&gt;http://example.com/\[\&lt;/a&gt;&lt;/p&gt;
    </code></pre>
    </div>
    </div>
    <p>An [email autolink]: #email-autolink
    [email address]: #email-address
    [tag name]: #tag-name
    [attribute]: #attribute
    [attribute name]: #attribute-name
    [attribute value specification]: #attribute-value-specification
    [attribute value]: #attribute-value
    [unquoted attribute value]: #unquoted-attribute-value
    [single-quoted attribute value]: #single-quoted-attribute-value
    [double-quoted attribute value]: #double-quoted-attribute-value
    [open tag]: #open-tag
    [closing tag]: #closing-tag
    [HTML comment]: #html-comment
    [processing instruction]: #processing-instruction
    [declaration]: #declaration
    [CDATA section]: #cdata-section
    [HTML tag]: #html-tag
    [hard line break]: #hard-line-break
    [<code>or</code>![`</p>
    <p>we insert a text node with these symbols as its literal content, and we
    add a pointer to this text node to the [delimiter stack]: #delimiter-stack</p>
    
    </body>
    </html>

The first line and the last two lines are given for context. The rest probably doesn't belong here. I'll leave it to you to work out where this came from, since I couldn't get lua to work on my system (using the current HEAD e34efc695057f8af102ff59eefac81efdb50382a):

```
lua tools/make_spec.lua html < spec.txt > spec.html
lua: ./lcmark.lua:59: attempt to index local 'defaultEnv' (a nil value)
stack traceback:
	./lcmark.lua:59: in main chunk
	[C]: in function 'require'
	tools/make_spec.lua:1: in main chunk
	[C]: ?
Makefile:13: recipe for target 'spec.html' failed
make: *** [spec.html] Error 1
```

**Comment by jgm** [comment]
Yes, this has already been fixed in the dev version (I've moved to
a much more robust script for creating spec.html that uses
the cmark API).  It will be fixed in the next point release.


+++ Martin von Gagern [Jan 12 16 08:27 ]:

**Comment by jgm** [comment]
Fixed by 0.24.
